src/pages/workspaces/workspace/analysis/modals/CromwellModal.js

```diff
@@ -65,25 +69,24 @@ export const CromwellModalBase = withDisplayName('CromwellModal')(
     });
 
     const renderActionButton = () => {
-      return !app ? h(ButtonPrimary, { onClick: createCromwell }, ['Create']) : h(ButtonPrimary, { onClick: onDismiss }, ['Close']);
-    };
-
-    const getEnvMessageBasedOnStatus = (app) => {
-      const waitMessage = 'This process will take up to a few minutes.';
-      const nonStatusSpecificMessage = 'Create a Cromwell environment to run workflows in your current workspace?';
-
+      const cookieReady = Utils.cond(
+        [cloudProvider === cloudProviderTypes.AZURE, () => azureCookieReady.readyForCromwellApp],
+        [Utils.DEFAULT, () => leoCookieReady]
+      );
       return !app
-        ? nonStatusSpecificMessage
-        : Utils.switchCase(
-            app.status,
-            ['PROVISIONING', () => 'The cloud compute is provisioning, which may take several minutes.'],
-            ['STOPPED', () => 'The cloud compute is paused.'],
-            ['PRESTOPPING', () => 'The cloud compute is preparing to pause.'],
-            ['STOPPING', () => `The cloud compute is pausing. ${waitMessage}`],
-            ['PRESTARTING', () => 'The cloud compute is preparing to resume.'],
-            ['STARTING', () => `The cloud compute is resuming. ${waitMessage}`],
-            ['RUNNING', () => 'There is already an instance of Cromwell running.'],
-            ['ERROR', () => 'An error has occurred on your cloud environment.']
+        ? h(ButtonPrimary, { onClick: createCromwell }, ['Create'])
+        : h(
+            ButtonPrimary,
+            {
+              href: app?.proxyUrls['cbas-ui'],
+              disabled: !cookieReady,
+              tooltip: Utils.cond([cookieReady, () => 'Open'], [Utils.DEFAULT, () => 'Please wait until Cromwell is running']),
+              onClick: () => {
+                Ajax().Metrics.captureEvent(Events.applicationLaunch, { app: appTools.CROMWELL.label });
+              },
+              ...Utils.newTabLinkPropsWithReferrer,
+            },
+            ['Open Cromwell']
```

Comment:
Just trying to make the experience more consistent across app types. Before, if you had a running Cromwell you would see this modal:
<img width="664" alt="image" src="https://github.com/DataBiosphere/terra-ui/assets/5368863/385a17f9-eb9e-4e36-bc4d-84097c3da779">

Now you have an opportunity to open it:
<img width="675" alt="image" src="https://github.com/DataBiosphere/terra-ui/assets/5368863/eb885831-61f1-4b6b-8f6b-8ef99db1709b">